governance-exercises/Le Bateleur - Poll Exercise 01/Forum Response.md

```diff
@@ -0,0 +1,45 @@
+# Poll Assignment 01
+
+### My Forum Responses
+
+---
+
+Response #1
```

Comment:
This should likely be dealt with in private first, to ensure you're not starting a messy public disagreement. 

In an ideal world, everyone would be fine with deal with the issues in a public and transparent manner. In reality, humans can get very defensive, very quickly.

Reply:
In practice, we've tended to deal with stuff like this by having the other stakeholders edit their posts to include required information. If anyone consistently made the same mistake, I would likely start doing it in public, but generally they just start doing the right things after you correct them privately a couple of times.